code/modules/research/designs/weapons.dm

```diff
@@ -1,7 +1,6 @@
 /datum/design/item/weapon
 	category = "Weapons"
 	price = 7000
-	build_type = STORE | PROTOLATHE
```

Comment:
this was intentional, please put it back. All weapons are meant to be available from both